requirements/requirements.txt

```diff
@@ -98,4 +98,5 @@ email_validator==1.0.1
 dnspython==1.13.0
 raven==6.0.0
 quickcache==0.0.1
+git+https://github.com/mvantellingen/python-zeep.git@46653fac2a92a99bc93d20c772cd7d797c301105
```

Comment:
This is the same version that successfully worked when i used zeep for the integration i did earlier. So just used the trusted version. Checking with latest version now.